dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/JacksonEmptyContainerLoader.java

```diff
@@ -85,22 +84,9 @@ private Optional<Object> constructEmptyInstance(Type type, TypeMarker<?> origina
             }
         }
 
-        Constructor<?> emptyRecordConstructor = getEmptyRecordCanonicalConstructor(type);
-        if (emptyRecordConstructor != null) {
-            try {
-                return Optional.of(emptyRecordConstructor.newInstance());
-            } catch (InstantiationException | IllegalAccessException | InvocationTargetException e) {
-                if (log.isDebugEnabled()) {
-                    log.debug("Empty record construction failed", SafeArg.of("type", type), e);
-                }
-                return Optional.empty();
-            }
-        }
```

Comment:
Should we maybe keep this piece of code around? (I'm not sure exactly in which case we needed it, but it seems potentially separately useful from the rest of the original PR itself)